client/components/ImageInSlide.jsx

```diff
@@ -0,0 +1,11 @@
+
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
```

Comment:
No unused imports (unless you plan to use them later, should only need to import "React from 'react'" for these most of your components.

Reply:
Good catch @MicahDShackelford , I copied and pasted them from index.jsx and forgot to remove ReactDom imports :)
It's been fixed now.
Thanks,